src/test/protocol/types_test.cpp

```diff
@@ -30,6 +30,11 @@ struct types_test : public beast::unit_test::suite
         auto const s = "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh";
         if (BEAST_EXPECT(parseBase58<AccountID>(s)))
             BEAST_EXPECT(toBase58(*parseBase58<AccountID>(s)) == s);
+        {
```

Comment:
[nit] For me the intention is not very clear - is this a deliberate scope introduction or a mistake in using `if` statement. To make it clearer I would mark the borders of the scope of `if` explicitly to make clearly separate for the new scope. 

`
if (BEAST_EXPECT(parseBase58<AccountID>(s))) {
    BEAST_EXPECT(toBase58(*parseBase58<AccountID>(s)) == s);
}

{
    auto const s =
        "âabcd1rNxp4h8apvRis6mJf9Sh8C6iRxfrDWNâabcdAVâ\xc2\x80\xc2\x8f";
    BEAST_EXPECT(!parseBase58<AccountID>(s));
}
`

Reply:
Yeah, now that you mention it, it does look a little off. I changed the test a little bit to make that all clearer, and to remove the need to parse `s` twice.